elytron/ELY-2544-jaas-realm-via-custom-realm.adoc

```diff
@@ -0,0 +1,100 @@
+= WFLY-18163 Make it possible to use JaasSecurityRealm via a custom-realm resource
```

Comment:
@Skyllarr Please add jekyll front matter above this line so it shows up in the appropriate categories on https://docs.wildfly.org/wildfly-proposals/

Categories are shown in https://github.com/wildfly/wildfly-proposals/blob/main/_data/wildfly-categories.yaml

The first four lines in https://raw.githubusercontent.com/wildfly/wildfly-proposals/main/security/WFCORE-3873_SNI_Support.adoc are an example of what I mean. Almost all currently merged proposals are also examples.

Please ping me in zulip when done and I'll merge this, as the feature itself is merged. Woohoo!

Reply:
@bstansberry I added the categories on top, thank you!